src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlSecurityTokenHandler.ValidateToken.Internal.cs

```diff
@@ -61,13 +61,27 @@ internal async Task<ValidationResult<ValidatedToken>> ValidateTokenAsync(
                 return issuerValidationResult.UnwrapError().AddStackFrame(StackFrames.IssuerValidationFailed);
             }
 
-            var signatureValidationResult = ValidateSignature(samlToken, validationParameters, callContext);
+            ValidationResult<SecurityKey> signatureValidationResult = ValidateSignature(samlToken, validationParameters, callContext);
+
             if (!signatureValidationResult.IsValid)
             {
                 StackFrames.SignatureValidationFailed ??= new StackFrame(true);
                 return signatureValidationResult.UnwrapError().AddStackFrame(StackFrames.SignatureValidationFailed);
             }
 
+            ValidationResult<ValidatedSigningKeyLifetime> issuerSigningKeyValidationResult = validationParameters.IssuerSigningKeyValidator(
```

Comment:
Does IssuerSigningKeyValidationDelegate need the BaseConfiguration? configuration parameter?
ValidationParameters has a ConfigurationProperty.

Reply:
In the JWT security handler we use base configuration and update it as needed. In the SAML and SAML2 ones the configuration is "merged" into ValidationParameters at the beginning, so there is no point in passing it around along the parameters.

Reply:
We do need to keep the `BaseConfiguration` in the delegate for the JWT cases, and potentially for the future case where SAML aligns with that behaviour.